packages/shared/src/utils/errors.ts

```diff
@@ -0,0 +1,15 @@
+// When API error codes are available, replace this substring check with a proper error code check
```

Comment:
Good initiative!

Reply:
This message has been there for a long long time 😅